libenv/sysinfo.c

```diff
@@ -972,7 +972,7 @@ static void Get3Environment(EvalContext *ctx)
 
         if (*context == '@')
         {
-            if (sscanf(context + 1, "%[^=]=%[^\n]", name, value) == 2)
+            if (sscanf(context + 1, "%" STRING(CF_MAXVARSIZE - 1) "[^=]=%" STRING(CF_BUFSIZE - 1) "[^\n]", name, value) == 2)
```

Comment:
Other part of this file, for instance a bit below on line 997, use hardcoded numbers instead. Should I unify everywhere?

Reply:
I wouldn't worry about that, yet.